src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -147,6 +149,15 @@ private SwiftCompileStep makeCompileStep() {
       compilerCommand.add(
           "-import-objc-header",
           getResolver().getRelativePath(bridgingHeader.get()).toString());
+
+      // bridging header needs exported headers for imports
+      Path headerPath = CxxDescriptionEnhancer.getHeaderSymlinkTreePath(
+          getProjectFilesystem(),
+          BuildTargets.unflavored(getBuildTarget()),
```

Comment:
Yeah. Doing it at the moment. But need to refactor dependencies a bit because I need to filter flavors from `AppleDebugFormat` as well.

This is my additional refactoring https://github.com/facebook/buck/pull/916